app/src/main/java/io/gnosis/safe/ui/safe/balances/SafeBalancesFragment.kt

```diff
@@ -27,9 +35,24 @@ class SafeBalancesFragment : SafeOverviewBaseFragment<FragmentSafeBalancesBindin
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
+        with(binding) {
+            val adapter = BalancesPagerAdapter(this@SafeBalancesFragment)
+            balancesContent.adapter = adapter
+            TabLayoutMediator(balancesTabBar, balancesContent, true) { tab, position ->
+                if (position % 2 == 0) {
```

Comment:
I suggest having enum Tab or Page { COINS, COLLECTABLES } and using this enum in mediator
when (Tab.values()[position]) {
   Tab.COINS ->
   Tab.COLLECTABLES ->
...
this will make it more readable